app/config.py.example

```diff
@@ -1,7 +1,7 @@
+# flake8: noqa
```

Comment:
Remove this and split `WEBHOOK_URL` into multiple lines comment

Reply:
I see you've changed it :)

Generally, prefer putting NOQA only on one line, and to specify the error code you want to ignore